OpenSim/Simulation/Model/Model.cpp

```diff
@@ -1686,7 +1686,7 @@ void Model::writeMarkerFile(const string& aFileName)
  * @param aMarkerSet The new marker set to copy.
  * @return Number of markers that were successfully added to the model.
  */
-int Model::replaceMarkerSet(const SimTK::State& s, const MarkerSet& aMarkerSet)
+int Model::replaceMarkerSet(SimTK::State& s, const MarkerSet& aMarkerSet)
```

Comment:
The state as an argument seems completely unnecessary for this method. I would prefer it was removed altogether. This should be considered a properties edit `initSystem()` should not be invoked in here.